apps/functional_chat/package.json

```diff
@@ -78,5 +79,6 @@
       "tailwindcss": {},
       "autoprefixer": {}
     }
-  }
+  },
+  "packageManager": "yarn@1.22.19+sha1.4ba7fc5c6e704fce2066ecbfb0b0d8976fe62447"
```

Comment:
Lets just stick to npm for now. No reason to introduce another package manager. For running instructions, see https://github.com/fw-ai/forge/tree/main/apps/functional_chat